python/paddle/fluid/backward.py

```diff
@@ -1101,67 +1102,119 @@ def append_backward(loss,
         isinstance(callbacks, list)
 
     program = loss.block.program
+
+    # Todo(liym27): when to add 1 in control flow
```

Comment:
Could you fully explain the comment? Did you mean when to add 1 in sub-block (control flow)?

Reply:
Yes. `program._appending_grad_times` is used for double grad which is not supported if `minimize/append_backward` is in sub-block (control flow)